samples/winapi_filefind/main.c

```diff
@@ -0,0 +1,55 @@
+#include <stdio.h>
+#include <fileapi.h>
```

Comment:
Is fileapi.h a standard Windows name?

We should attempt to reach more XDK / Windows code compatibility in the future.

*I know that the XDK often uses a catchall `xbox.h`; which would probably be recreated in include/xdk/xbox.h in the future.*

Reply:
Yes, according to [Microsoft's online docs](https://docs.microsoft.com/en-us/windows/desktop/api/fileapi/nf-fileapi-findfirstfilea), fileapi.h is the correct header name. I tried to put everything exactly where it belongs to ensure maximum compatibility with Windows (using Microsoft and ReactOS/wine docs as a reference).

Microsoft does recommend to just include windows.h though, imho we should definitely add that header in the future.

Reply:
Excellent! Thanks for confirming.